lib/core/errors.ts

```diff
@@ -1,6 +1,8 @@
 export enum ErrorCodes {
```

Comment:
The KILLED error code is replaced by the abort so we should be able to remove it and repurpose the 30s, while we're on v0 I think it's safe to move these codes around